SubEthaEdit-Mac/Modes/Javascript.seemode/Contents/Resources/ModeSettings.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings>
     <recognition>
-        <extension>js</extension>
```

Comment:
The order matters here, the first extension is the one used for new files, so we need to keep `.js` at the top here.